plugins-compat-tester/src/main/java/org/jenkins/tools/test/maven/ExternalMavenRunner.java

```diff
@@ -24,6 +24,8 @@
  */
 public class ExternalMavenRunner implements MavenRunner {
 
+    public static final String DISABLE_DOWNLOAD_LOGS = "-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn";
```

Comment:
isn't `--no-transfer-progress` / `-ntp` the preferred option for this now? not sure if you can pass an option through here though

Reply:
We do not set high Maven requirements in the tool so far. `-ntp` option would fail in the current Dockerfile, for example: https://github.com/jenkinsci/plugin-compat-tester/blob/master/Dockerfile

Reply:
probably no need for this to be public? even having it as a constant is debate-able?

```suggestion
    private static final String DISABLE_DOWNLOAD_LOGS = "-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn";
```

Reply:
I agree, i'll change it to private.